src/notifications/screens/notifications.screen.js

```diff
@@ -196,15 +234,53 @@ class Notifications extends Component {
     });
   };
 
-  navigateToThread(notification) {
-    const { markAsRead, navigation } = this.props;
+  saveContentBlockHeight = e => {
+    const { height } = e.nativeEvent.layout;
 
-    markAsRead(notification.id);
-    navigation.navigate('Issue', {
-      issueURL: notification.subject.url.replace('pulls', 'issues'),
-      isPR: notification.subject.type === 'PullRequest',
-      language: this.props.language,
-    });
+    this.setState({ contentBlockHeight: height - sizes.tabBar });
+  };
+
+  keyExtractor = (item, index) => {
+    return index;
+  };
+
+  isLoading() {
+    const {
+      unread,
+      participating,
+      all,
+      isPendingUnread,
+      isPendingParticipating,
+      isPendingAll,
+    } = this.props;
+    const { type } = this.state;
+
+    switch (type) {
+      case 0:
+        return unread && isPendingUnread;
+      case 1:
+        return participating && isPendingParticipating;
+      case 2:
+        return all && isPendingAll;
+      default:
+        return null;
+    }
+  }
+
+  notifications() {
+    const { unread, participating, all } = this.props;
+    const { type } = this.state;
+
+    switch (type) {
+      case 0:
+        return unread;
+      case 1:
+        return participating;
+      case 2:
+        return all;
+      default:
+        return null;
```

Comment:
Hey, looks like `#notifications` method should always return an array, it is safer in `default` return also an array( probably [])

I know that default cause should be not called, but I believe it's always nice to try keep the `type return` consistency 💃